tests/jballerina-unit-test/src/test/resources/test-src/statements/expression/expression-stmt-negative.bal

```diff
@@ -15,3 +15,18 @@ function test2(){
     bar();
     x;
 }
+
+function baz() returns error|int|() => ();
```

Comment:
:thinking: Let's change the test case to reflect and NPE.
:pushpin: Line number 19 in the test case masks the NPE.